code/GPRs/translateGrRules.m

```diff
@@ -117,6 +117,9 @@
 
 % allow for non-exact matches to "Symbol" option
 targetFormat(ismember(lower(targetFormat),{'name','names','symbol','symbols'})) = {'Symbol'};
+if ismember(lower(gene_type_orig), {'name','names','symbol','symbols'})
```

Comment:
Hi @JonathanRob, I saw you have shortly explained this change in the PR #331 conversation tab, I was wondering if this change is connected to any issue that could explain the idea behind it? thanks

Reply:
Good question @Rasools. It's not connected to any particular issue. It's just based on my own experience using the function, since I never remember whether it's "name" or "symbol" or pluralized, etc. so I assumed others might run into a similar issue.

Reply:
Great! thanks for adding this.